src/components/product.js

```diff
@@ -128,22 +128,31 @@ export default class Product extends Component {
       imageSize = parseInt(this.options.width, 10) || 480;
     }
 
-    let src;
     let id;
+    let src;
+    let srcLarge;
+    const MODAL_IMAGE_SIZE = 550;
```

Comment:
maybe define this at the top of the function?

Would also be nice to define that `480` default from above as a constant along with it.